staging/src/k8s.io/api/imagepolicy/v1alpha1/types.go

```diff
@@ -68,6 +69,7 @@ type ImageReviewContainerSpec struct {
 // ImageReviewStatus is the result of the review for the pod creation request.
 type ImageReviewStatus struct {
 	// Allowed indicates that all images were allowed to be run.
+	// +optional
```

Comment:
@ShaanveerS Have you found the validation code for this API? It's not where I expected it to be and wanted to check the validations. How did you come to this?